spec/lib/sufia/export/collection_converter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Sufia::Export::CollectionConverter do
+  let(:collection) do
+    Collection.create(title: "title1", creator: ["creator1"], description: "description1") do |col|
```

Comment:
Yes, I do believe they should, according to https://github.com/projecthydra/sufia/blob/master/spec/factories/collections_factory.rb#L23-L26

Would it be possible to use a factory here instead of creating a repository object? That'd help with suite speed.